server/app/views/ViewUtils.java

```diff
@@ -370,4 +375,99 @@ public static DivTag makeAlert(String text, Optional<String> maybeTitle, String.
   public static DivTag makeAlertInfoSlim(String text) {
     return makeAlert(text, Optional.empty(), BaseStyles.ALERT_INFO, BaseStyles.ALERT_SLIM);
   }
+
+  /**
+   * Makes a USWDS Modal component that opens via a button click.
+   * https://designsystem.digital.gov/components/modal/
+   *
+   * @param body The HTML element that will be the main content of the modal.
+   * @param elementIdPrefix The prefix for the HTML element ids.
+   * @param headerText The header text for the modal.
+   * @param linkButtonText The text that will be on the button that opens the modal.
+   * @param hasFooter A boolean value that determines whether to include a footer with action
+   *     buttons. If the main content has a form, the buttons will already be included with the
+   *     form, so no need for the footer.
+   * @param firstButtonText Text for the first footer button.
+   * @param secondButtonText Text for the second footer button.
+   * @return DivTag containing the button that opens the modal and the modal itself.
+   */
+  public static DivTag makeUSWDSModal(
+      ContainerTag body,
```

Comment:
Melanie- Can the container body by a FormTag?

Reply:
Yes, absolutely.  If you check my draft PR [here](https://github.com/civiform/civiform/pull/6182/files#diff-7a8defb1c6ec25bcdafaf12f0a97988273796eec3a4f9d15e17ca30e21e4036b) you'll see that's what I'm passing in for the new client modal.